pkg/sql/schemachange/alter_column_type.go

```diff
@@ -101,11 +101,11 @@ var classifiers = map[types.Family]map[types.Family]classifier{
 		types.BytesFamily: func(s *types.T, b *types.T) ColumnConversionKind {
 			switch {
 			case b.Width() == 0:
-				return ColumnConversionTrivial
+				return ColumnConversionValidate
```

Comment:
Can't this whole function just return `ColumnConversionValidate` now?

Reply:
Done